packages/wrangler/src/cloudchamber/build.ts

```diff
@@ -146,14 +166,20 @@ export async function buildAndMaybePush(
 					`Untagging built image: ${imageTag} since there was no change.`
 				);
 				await runDockerCmd(pathToDocker, ["image", "rm", imageTag]);
-				return "";
+				return { image: digests[0], pushed: false };
 			} catch (error) {
 				logger.log(`Image does not exist remotely, pushing: ${imageTag}`);
+				if (error instanceof Error) {
+					logger.debug(
```

Comment:
how important are these errors for users? are we sure `debug` is the log level we want here? This is not the default log level in wrangler, so it might be a lot of folks will miss out on this info, unless they specifically set the log level to debug. Totally your decision, just thinking out loud